### DISCUSSION TITLE
Can't get it to run on debian

### DISCUSSION THREAD

**Discussion by catty-dev** [alfiankan/teleterm#20] [Q&A] [answered]
The bot worked fine on my ubuntu vps but now I'm using debian 11 and cant get it to work.

debian@vps:~$ teleterm-bin
-bash: /usr/local/bin/teleterm-bin: Permission denied
debian@vps:~$ sudo teleterm-bin
sudo: teleterm-bin: command not found

Any suggestions how to fix that?

**Comment by alfiankan** [comment]
it might be the binary not moved/copied properly. Please reproduce instalation steps here

Like
1. wget https://github.com/alfiankan/teleterm/releases/download/v2.1.0/teleterm-v2.1.0-linux-amd64.tar.gz
2. tar -xf teleterm-v2.1.0-linux-amd64.tar.gz
3. ...
4. ...
5. ...

**Comment by alfiankan** [comment, marked as answer]
or you can try this installation step using prebuilt binary, this steps is working on my debian machine :

1. installing wget `sudo apt install wget`
2. if you're using amd64 download prebuild binary for linux amd64 `wget https://github.com/alfiankan/teleterm/releases/download/v2.1.0/teleterm-v2.1.0-linux-amd64.tar.gz`
3. if you're using arm64 download prebuild binary for linux arm64 `wget https://github.com/alfiankan/teleterm/releases/download/v2.1.0/teleterm-v2.1.0-linux-arm64.tar.gz`
4. extract using tar `tar -xf teleterm-v2.1.0-linux-amd64.tar.gz` or `tar -xf teleterm-v2.1.0-linux-arm64.tar.gz`  for arm64
5. copy to `/usr/local/bin` using `sudo cp teleterm /usr/local/bin/`
6. running fresh setup `teleterm fresh`
7. configure config file at `/home/<username>/.teleterm/config.yaml` put your telegram bot token there
8. run the bot `teleterm`

spoiler :
<img width="799" alt="Screenshot 2023-01-01 at 04 39 38" src="https://user-images.githubusercontent.com/40946917/210156089-062ccd5b-4c55-4452-80a5-2fa6ebd8293f.png">

Happy new year 2023 🎉
